test/libsolidity/syntaxTests/nameAndTypeResolution/005_type_inference_smoke_test.sol

```diff
@@ -1,8 +1,7 @@
 contract test {
     function f(uint256 arg1, uint32 arg2) public returns (bool ret) {
-        var x = arg1 + arg2 == 8; ret = x;
+        bool x = arg1 + arg2 == 8; ret = x;
```

Comment:
This is definitely testing type deduction and isn't that useful anymore.

Reply:
I will remove it then. thanks!

Reply:
Well if there's no other test for inferring bool then this could stay. Lets keep it.

Reply:
IMHO, "type inference" (name of the test) means assigning a non-explicit type to a variable, and not to an expression, so I think the test should be renamed.